README.md

```diff
@@ -1,2 +1,3 @@
-# triggers
-Event triggering with Tekton!
+# Tekton Triggers
+
+[Tekton](https://github.com/tektoncd/pipeline) leverages Kubernetes enabling you to create containerized, composable, and configurable workloads in a **Cloud Native** way. Although, Tekton simplifies the creation of workloads, it is unopinionated on how to generate necessary resources (notably, [PipelineRuns](https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md) and [PipelineResources](https://github.com/tektoncd/pipeline/blob/master/docs/resources.md#pipelineresources)). `tektoncd/triggers` reconciles this data transformation layer by processing event payloads into Kubernetes resources through the `TriggerTemplate` and `TriggerBinding` CRDs. Using `tektoncd/triggers` in conjunction with `tektoncd/pipeline` enables you to easily create continuous integration and delivery (CI/CD) systems where the execution is defined **entirely** through Kubernetes resources. This repo draws inspiration from `Tekton`, but can used stand alone since `TriggerTemplates` can create any Kubernetes resource. 
```

Comment:
usually for markdown in the other repos we wrap the lines, tho i couldn't tell you exactly how many chars (~79? haha)

Reply:
For sure, it was a bit bulky as is.